src/examples/echo/echo.go

```diff
@@ -0,0 +1,41 @@
+// This is a echo console running on the device UART.
+// Connect using 57600 baud, 8-N-1 with your terminal program.
+package main
+
+import (
+	"machine"
+	"time"
+)
+
+func main() {
+	// Set baudrate to 56k for an Arduino Uno to be error-free.
+	machine.UART0.Configure(machine.UARTConfig{Baudrate: 57600})
+	machine.UART0.Write([]byte("Echo console enabled. Type something then press enter:\r\n"))
+
+	input := make([]byte, 64)
+	i := 0
+	for {
+		if machine.UART0.Buffered() > 0 {
+			data, _ := machine.UART0.ReadByte()
+
+			// Remove high-order bit because 7-bit ascii
+			data &^= 0x80
```

Comment:
It works just fine for me when I remove this line (even at 57600 baud rate). Are you sure it is still present?